.eslintrc.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
 
     // Temporary overrides. Logic to be rewritten.
     // TODO https://github.com/apiaryio/gavel.js/issues/150
-    'no-param-reassign': 'off'
+    'no-param-reassign': 'off',
+    'no-unused-expressions': 'off'
```

Comment:
```js
expect(obj).to.be.valid
```
is considered a violation of this rule. Therefore, disabling the rule. Such violations are not present anywhere else in the code.

Reply:
I think this should be then a separate "section" if the ignores:

```js
... 

// Disabled to allow expect() expressions in tests
'no-unused-expressions': 'off',

// Temporary overrides. Logic to be rewritten
'no-param-reassign': 'off',
...
```

Reply:
Grouped in cbfb18c2dbb9856f00e7458dcd6f940d332168e0.